docs/en/observability/index.asciidoc

```diff
@@ -61,6 +61,8 @@ include::logs-overview.asciidoc[leveloffset=+1]
 
```

Comment:
Don't forget to update `logs-overview.asciidoc` to include a snippet on this new documentation ([link](https://observability-docs_3074.docs-preview.app.elstc.co/guide/en/observability/master/logs-observability-overview.html))